app/tests/communication/test_banner_integration.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import status
+
+import pytest
+
+from app.communication.factories.banner_factory import BannerFactory
+
+API_BANNER_URL = "/banners/"
+
+API_VISIBLE_BANNER_URL = "/banners/visible/"
+
+
+@pytest.mark.django_db
+def test_get_banners_as_anonymus_user(default_client):
+    "An anonymus user is not able to list all banners"
+    url = API_BANNER_URL
+
+    response = default_client.get(url)
```

Comment:
Tenker du kan legge `API_BANNER_URL` rett inn i `get`, samme gjelder de andre testene
```suggestion
    response = default_client.get(API_BANNER_URL)
```